devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -62,8 +63,18 @@ def parse(String description) {
         sendEvent(event)
     }
     else {
-        log.warn "DID NOT PARSE MESSAGE for description : $description"
-        log.debug zigbee.parseDescriptionAsMap(description)
+        def cluster = zigbee.parse(description)
```

Comment:
Is this portion necessary if we're not setting the customized `onOffConfig()` interval?